cmd/dm-worker/main.go

```diff
@@ -52,6 +52,7 @@ func main() {
 		Format: cfg.LogFormat,
 		Level:  strings.ToLower(cfg.LogLevel),
 	})
+	log.SetRedactLog(cfg.RedactInfoLog)
```

Comment:
Should we maybe add a message so it easy to confirm that it is indeed enabled?

I assume it isn't supported to change this on a running instance? otherwise we should probably log in `SetRedactLog()`.

Reply:
yes! let me add a log item here